helm-chart/templates/ingress.yaml

```diff
@@ -0,0 +1,61 @@
+{{- if .Values.ingress.enabled -}}
```

Comment:
this whole template can be removed.. and the `NOTES.txt` will need to be updated accordingly